lib/domain-shared/shared_use_cases/get_words_from_private_key_use_case.dart

```diff
@@ -1,28 +1,26 @@
 import 'package:seeds/datasource/local/settings_storage.dart';
 import 'package:seeds/screens/authentication/import_key/interactor/usecases/generate_key_from_recovery_words_use_case.dart';
+import 'package:seeds/screens/authentication/import_key/interactor/usecases/generate_key_from_seeds_passport_words_use_case.dart';
 
 class GetWordsFromPrivateKey {
   /// If the private key has words, it returns the list of words
-  /// separated by a dash (-), otherwise an empty list.
+  /// otherwise an empty list.
+  /// Handles both passport and seeds light wallet style 12 words keys
   List<String> run() {
     final String privateKey = settingsStorage.privateKey!;
 
-    ///-------IS PRIVATE KEY 12 WORDS TYPE
-    // Convert all words into private keys
-    final List<String> privateKeysListFromWords = settingsStorage.recoveryWords
-        .map((i) => GenerateKeyFromRecoveryWordsUseCase().run(i.split('-')))
-        .toList()
-        .map((i) => '${i.eOSPrivateKey}')
-        .toList();
-    // Verify if there is a match with the selected private key
-    final bool is12WordsAccount = privateKeysListFromWords.contains(privateKey);
-    if (is12WordsAccount) {
-      final int wordsIndex = privateKeysListFromWords.indexOf(privateKey);
-      // Since the storage recover words and privateKeysListFromWords are map 1:1
-      // the index will be the same to get the target words
-      return settingsStorage.recoveryWords[wordsIndex].split('-');
-    } else {
-      return [];
-    }
+    final wordsString = settingsStorage.recoveryWords.firstWhere((item) {
+      final words = item.toWordList();
+      return GenerateKeyFromRecoveryWordsUseCase().run(words).eOSPrivateKey.toString() == privateKey ||
+          GenerateKeyFromSeedsPassportWordsUseCase().run(words).eOSPrivateKey.toString() == privateKey;
+    }, orElse: () => "");
```

Comment:
''

Reply:
is '' better than "" ?? 

Always kinda confused by this.

Reply:
Done

Reply:
also a dart thing, 
At some point we changed every single "" in our app for ''